vimeo/vimeo-networking-java
# MPS-20: Adding Serializable support to v2 models

#### Issue
[MPS-20](https://vimean.atlassian.net/browse/MPS-20)


#### Summary
Updated all the v2 models to implement `Serializable`. 

#### How to Test
I wrote some test files in the main app to make a network request and pass along the model to a new activity where I used `intent.getSerializableExtra` to confirm the model serialized correctly.  

Let me know if anyone wants me to supply the test files. 👍
